email/theme.properties

```diff
@@ -0,0 +1,23 @@
+# for keycloak - don't edit keys
+locales=en
+
+# custom
+
+## images
+bannerVerify=ohcrn-email_banner-verify.jpg
+baseImageUrl=http://localhost:8087/img/
```

Comment:
One more thing we should bring up tomorrow during planning: we should create a ticket to track the work of putting all the email-related images somewhere in the main OHCRN site's assets, and then updating these `baseImageUrl`s

Reply:
https://github.com/OHCRN/platform/issues/1031